### DISCUSSION TITLE
Pi Zero W Limitations on reed sensors...

### DISCUSSION THREAD

**Discussion by Menethoran** [mKeRix/room-assistant#444] [General]
Anyone know if theres a bandwidth limitation on a pi zero on the number of switches it's capable of driving?

Im looking to add a bunch of reed sensors to doors/windows etc... and realized, i could probably run several off of a single Pi Zero (that already uses 1 motion sensor... and id actually like to add 1 or 2 more motion sensors to a couple of the Pis... so 1 could in effect, monitor 2 or more rooms if there is a central point shared amongst them (Like, a bedroom, bathroom, hallway... all sharing a single common corner... drill little holes to run the PIR sensors through and assign the GPIO to different sensors))...

Anyway, does anyone know the limits on how fast/well the Pi Zero W can handle this kind of setup if any?

**Comment by mKeRix** [comment]
I'm not really an expert on this topic, but from my limited understanding you can probably run quite a few of these off a single Pi Zero. I don't think they use a lot of power and Pis have plenty of available GPIO pins that you can hook up to. It should also perform quite well, at least I'm not aware of any issues between Pis and these kinds of simple GPIO sensors. :)
